html/includes/table/graylog.inc.php

```diff
@@ -36,7 +36,11 @@
         $query .= ' && ';
     }
     $ip = gethostbyname($filter_hostname);
+    $device = device_by_name($filter_hostname);
     $query .= 'source:"'.$filter_hostname.'" || source:"'.$ip.'"';
+    if (isset($device['ip'])) {
```

Comment:
Would it be worth adding a check that `$ip != $device['ip']` before adding the additional parameter?